website/www/site/assets/scss/_quotes.scss

```diff
@@ -107,7 +106,9 @@
       span {
         padding-right: 10px;
       }
+
```

Comment:
make sure to format every file you work with. It helps to increase code consistency and to avoid future merge conflicts.

Reply:
fixed